hw/ip/i2c/dv/i2c_sim_cfg.hjson

```diff
@@ -30,8 +30,7 @@
                 "{proj_root}/hw/dv/data/tests/csr_tests.hjson",
                 "{proj_root}/hw/dv/data/tests/intr_test.hjson",
                 "{proj_root}/hw/dv/data/tests/tl_access_tests.hjson"]
-                // TODO: enable stress tests later.
-                //"{proj_root}/hw/dv/data/tests/stress_tests.hjson"]
+                "{proj_root}/hw/dv/data/tests/stress_tests.hjson"]
```

Comment:
syntax error at line 32 due to ending with `]`
perhaps we keep this commented out until we have stress_all test

Reply:
I fixed syntax error but also commented out that line